manifests/controlled_resources_list.yaml

```diff
@@ -0,0 +1,77 @@
+resources:
+  - GroupVersionKind:
+      group: security.istio.io
+      version: v1beta1
+      kind: AuthorizationPolicy
+  - GroupVersionKind:
+      group: networking.istio.io
+      version: v1alpha3
+      kind: DestinationRule
+  - GroupVersionKind:
+      group: networking.istio.io
+      version: v1alpha3
+      kind: EnvoyFilter
+    ControlledList:
+      - name: "stats-filter-1.16"
+        namespace: "istio-system"
+      - name: "stats-filter-1.17"
+        namespace: "istio-system"
+      - name: "stats-filter-1.18"
+        namespace: "istio-system"
+      - name: "tcp-stats-filter-1.16"
+        namespace: "istio-system"
+      - name: "tcp-stats-filter-1.17"
+        namespace: "istio-system"
+      - name: "tcp-stats-filter-1.18"
```

Comment:
Yeah, I was also thinking about this one, but since we will be providing new version of operator for new Istio version it's not a technical problem. We might make it more flexible (like using regex and looking for `tcp-stats-filter-\d.\d\d"), but this might change in Istio resources so I don't have any strong opinion about this.
If you feel like it would be better, I will switch the match to use regex

Reply:
Changed the match to regex